migrations/2_alter_project_table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function (knex, Promise) {
+  return Promise.join(
+    knex.schema.table('projects', function (t) {
+      t.boolean('readonly');
```

Comment:
no? This is an alter, because that's how migrations work. In order to extend the already in production database with two new columns that we need from now on, you need to alter the schema is a second migration.